File: scripts/reset.mjs
import { execSync } from 'child_process';

try {
  console.log('[v0] Descartando cambios...');
  execSync('git reset --hard HEAD', { cwd: '/vercel/share/v0-project', stdio: 'inherit' });
  console.log('[v0] Cambios descartados exitosamente');
} catch (error) {
  console.error('[v0] Error:', error.message);
  process.exit(1);
}

File: scripts/reset.py
#!/usr/bin/env python3
import subprocess
import sys

try:
    # Descartar todos los cambios locales
    result = subprocess.run(['git', 'reset', '--hard', 'HEAD'], cwd='/vercel/share/v0-project', capture_output=True, text=True)
    print(result.stdout)
    if result.returncode != 0:
        print(f"Error: {result.stderr}", file=sys.stderr)
        sys.exit(1)
    print("[v0] Cambios descartados exitosamente")
except Exception as e:
    print(f"Error ejecutando git reset: {e}", file=sys.stderr)
    sys.exit(1)
